src/app/related-search/related-search.component.ts

```diff
@@ -52,7 +52,18 @@ export class RelatedSearchComponent implements OnInit {
       }
     });
   }
-
+  getQuery(q) {
+    return {query: q.toString(),
+      verify: false,
+      nav: 'filetype,protocol,hosts,authors,collections,namespace,topics,date',
+      start: 0,
+      indexof: 'off',
+      meanCount: '5',
```

Comment:
@nikhilrayaprolu I included it to maintain consistency in queries. Whether entered from search bar or by clicking on a link the query should look the same. If you wish, we can open a seperate issue, and remove meancount from queries